src/Hazelcast.Net.Tests/Remote/UnisocketTests.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Hazelcast.Core;
+using Hazelcast.Testing;
+using NUnit.Framework;
+
+namespace Hazelcast.Tests.Remote
+{
+    public class UnisocketTests : MultiMembersRemoteTestBase
+    {
+        [TearDown]
+        public async Task RemoveAllMembers()
+        {
+            foreach (var member in RcMembers)
+            {
+                await ShutdownMember(member.Key);
+            }
+        }
+        /// <summary>
+        /// Port of testClientListener_withDummyClient
+        /// </summary>
+        /// <returns></returns>
+        [Test]
+        public async Task TestEventsWithDummyClient()
```

Comment:
Java test name `testClientListener_withDummyClient`

Reply:
I mentioned to name on the description. Also, listeners mean events on .Net side, so tried to conserve it too.